src/cargo/sources/config.rs

```diff
@@ -125,6 +145,22 @@ impl<'gctx> SourceConfigMap<'gctx> {
         self.gctx
     }
 
+    fn load_and_overlay(
```

Comment:
nit: I tend to prefer putting internal functions after their use.  People tend to read top-down and tend to be looking at interfaces and then look into implementations where relevant.

Reply:
Done